core/plugins/taboo/taboo.py

```diff
@@ -11,11 +11,24 @@
 
 @ctx.action_class("user")
 class TabActions:
-    def tab_rename(name: str):
-        actions.user.vim_run_normal_exterm(f":TabooRename {name}")
+    def tab_rename(name: str = ""):
+        actions.user.vim_run_command_exterm(f":TabooRename {name}")
 
-    def tab_new_named(name: str = ""):
+    def tab_open_with_name(name: str = ""):
         actions.user.vim_run_command_exterm(f":TabooOpen {name}")
 
-    def tab_reset():
+    def tab_name_reset():
         actions.user.vim_run_command_exterm(":TabooReset")
+
+
+# FIXME: TEMPORARY: Once https://github.com/talonhub/community/pull/1446 is merged, this should be removed
+@mod.action_class
+class Actions:
+    def tab_rename(name: str = ""):
```

Comment:
I hadn't done `name: str = ""` in PR1446, since I'm always doing something like `tab rename [<user.text>]: user.tab_rename(text or "")`. 

So I think will break this eventually because the function signatures will be different.

That said... I guess it makes sense to just be able to call them without specifying the name to trigger the underlying rename function, so will update the community PR accordingly.

Reply:
fixed in community PR1446

Reply:
Also note I meant to comment on the context override method not this module one, since this one will go away.